contracts/todomanager.hpp

```diff
@@ -0,0 +1,51 @@
+#include <eosio/eosio.hpp>
+#include <eosio/system.hpp>
+#include <string>
+
+namespace todomanager
+{
+using namespace eosio;
+using std::string;
+
+class[[eosio::contract("todomanager")]] todomanager : public contract
+{
+public:
+    using contract::contract;
+
+    todomanager(name receiver, name code, datastream<const char *> ds) : contract(receiver, code, ds),
+                                                                         todos_table(receiver, code.value)
+    {
+    }
+
+    enum e_status_option : uint8_t {
```

Comment:
Rename it to status (what is the idea of e_status ?)

Reply:
updated enum type